_includes/footer.html

```diff
@@ -5,7 +5,7 @@
     <label>Ready to hand off the Gruntwork?</label>
   </div>
   <div>
-    <button class="btn btn-primary-rounded">Buy Now</button>
+    <a href="/contact" class="btn btn-primary-rounded" role="button" aria-pressed="true">Buy Now</a>
```

Comment:
The spacing and color of these elements look off. The label needs a lot more margin

Expected:
<img width="676" alt="Screen Shot 2019-09-05 at 14 11 40" src="https://user-images.githubusercontent.com/1516418/64352264-a1cbc780-cff3-11e9-801f-4680d5d927b7.png">

Current:
<img width="556" alt="Screen Shot 2019-09-05 at 15 40 34" src="https://user-images.githubusercontent.com/1516418/64352273-a55f4e80-cff3-11e9-80a4-88bd057f214a.png">